src/library-authoring/author-library/specs/LibraryAuthoringPage.spec.jsx

```diff
@@ -323,4 +326,23 @@ testSuite('<LibraryAuthoringPageContainer />', () => {
       () => expect(deleteLibraryBlock.fn).toHaveBeenCalledWith({ blockId: block.id }),
     );
   });
+
+  it('Rename library', async () => {
+    const library = libraryFactory();
+    const block = blockFactory(undefined, { library });
+    await render(library, genState(library, [block]));
+    
+    const editButton = screen.getByTestId('edit-name-button');
```

Comment:
More preferable to use `getByRole` instead `getByTestId`
`const editButton = screen.getByRole('button', { name: /Edit/i });` 
just update alt attr 'Edit' to something more specific, like `edit-name-button` and remove `data-testid`

Reply:
Updated